widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -137,4 +141,23 @@ public static void clearVisitorSession() {
         Dependencies.getControllerFactory().destroyControllers();
         Glia.clearVisitorSession();
     }
+
+    private static final Consumer<GliaException> updateVisitorExceptionConsumer = exception -> {
+        if (exception != null) {
+            Logger.e(TAG, "Visitor Info update failed: " + exception.toString());
+        } else {
+            Logger.e(TAG, "Visitor Info updated Success");
```

Comment:
The issue that @jakob-mc has mentioned is not resolved.
Please correct the logger level to debug.